src/pages/p2p/components/_availability.js

```diff
@@ -136,6 +136,7 @@ const Availability = () => {
                 mobile_margin="0 0 24px"
                 align="center"
                 as="h2"
+                type="page-title"
```

Comment:
This is handled in https://github.com/binary-com/deriv-com/pull/1386

Reply:
Ahh got it 👍